fcl/dunefd/reco/reco_supernova_dune10kt_1x2x2.fcl

```diff
@@ -0,0 +1,34 @@
+#include "standard_reco_dune10kt.fcl"
+
+services: 
+{
+    @table::services
+    @table::dunefd_1x2x2_reco_services
+}
+
+physics.producers:
+{
+    @table::physics.producers
+    wclsmcnfsp: @local::dune10kt_1x2x2_mc_nfsp
+}
+
+services.DetectorPropertiesService.NumberTimeSamples: 6000
+services.DetectorPropertiesService.ReadOutWindowSize: 6000
+
+physics.producers.gaushit.CalDataModuleLabel: "wclsmcnfsp:gauss"
+physics.producers.gaushit.HitFilterAlg.MinPulseHeight: [ 2, 2, 2 ]
+physics.producers.spsolve.MinNHits: 2 #lower the threshold for low energy interactions
+physics.producers.pmtracktc.WireModuleLabel: "wclsmcnfsp:gauss"
+
+physics.reco: [
```

Comment:
Can this reco path (and associated configurations) be defined in the workflow fhicl instead?

Reply:
@lpaulucc, ok I now define the SN workflow in workflow_reco_dune10kt.fcl and created a new standard_reco_dune10kt_supernova.fcl to further simplify things.